x-pack/legacy/plugins/ml/public/application/settings/calendars/edit/calendar_form/calendar_form.js

```diff
@@ -128,39 +133,59 @@ export const CalendarForm = ({
           </EuiFormRow>
         </Fragment>
       )}
-      {isEdit && <EditHeader calendarId={calendarId} description={description} />}
-      <EuiFormRow
-        label={
-          <FormattedMessage
-            id="xpack.ml.calendarsEdit.calendarForm.jobsLabel"
-            defaultMessage="Jobs"
-          />
-        }
-      >
-        <EuiComboBox
-          options={jobIds}
-          selectedOptions={selectedJobOptions}
-          onChange={onJobSelection}
-          isDisabled={saving === true || canCreateCalendar === false}
-        />
-      </EuiFormRow>
 
-      <EuiFormRow
+      <EuiSpacer size="m" />
```

Comment:
Nit - I think an `xl` spacer is better here, as we use above the Events section, as it more clearly groups the 'All groups' switch with the Jobs and Groups inputs.